web/src/components/storage/dasd/DASDTable.tsx

```diff
@@ -20,368 +20,562 @@
+  let emptyMode: DASDEmptyStateMode;
+  if (isEmpty(filteredDevices)) {
+    emptyMode = state.filters === initialState.filters ? "noDevices" : "noFilterResults";
+  }
+  /**
+   * Dispatches a DASD mutation and marks devices as waiting.
+   *
+   * @param mutation Parameters describing the DASD update operation
+   */
+  const updater = (mutation: DASDMutationFnProps) => {
+    updateDASD(mutation);
+    dispatch({ type: "START_WAITING", payload: mutation.devices });
   };
 
   return (
-    <>
-      <Toolbar>
-        <ToolbarContent>
-          <ToolbarGroup align={{ default: "alignEnd" }}>
-            <ToolbarItem>
-              <TextInputGroup>
-                <TextInputGroupMain
-                  value={minChannel}
-                  type="text"
-                  aria-label={_("Filter by min channel")}
-                  placeholder={_("Filter by min channel")}
-                  onChange={(_, minChannel) => updateFilter({ minChannel })}
-                />
-                {minChannel !== "" && (
-                  <TextInputGroupUtilities>
-                    <Button
-                      variant="plain"
-                      aria-label={_("Remove min channel filter")}
-                      onClick={() => updateFilter({ minChannel: "" })}
-                      icon={<Icon name="backspace" />}
-                    />
-                  </TextInputGroupUtilities>
-                )}
-              </TextInputGroup>
-            </ToolbarItem>
-            <ToolbarItem>
-              <TextInputGroup>
-                <TextInputGroupMain
-                  value={maxChannel}
-                  type="text"
-                  aria-label={_("Filter by max channel")}
-                  placeholder={_("Filter by max channel")}
-                  onChange={(_, maxChannel) => updateFilter({ maxChannel })}
-                />
-                {maxChannel !== "" && (
-                  <TextInputGroupUtilities>
-                    <Button
-                      variant="plain"
-                      aria-label={_("Remove max channel filter")}
-                      onClick={() => updateFilter({ maxChannel: "" })}
-                      icon={<Icon name="backspace" />}
-                    />
-                  </TextInputGroupUtilities>
-                )}
-              </TextInputGroup>
-            </ToolbarItem>
-
-            <ToolbarItem variant="separator" />
-
-            <ToolbarItem>
-              <Actions devices={selectedDASD} isDisabled={selectedDASD.length === 0} />
-            </ToolbarItem>
-          </ToolbarGroup>
-        </ToolbarContent>
-      </Toolbar>
-
-      <Page.Section aria-label={_("DASDs table section")}>
-        <PageContent />
-      </Page.Section>
-    </>
+    <Content>
+      {!isEmpty(state.waitingFor) && (
+        <Popup isOpen title={_("Applying changes")} disableFocusTrap>
+          <Content component="p">{_("Applying changes to the requested DASD devices.")}</Content>
```

Comment:
```suggestion
          <Content component="p" isEditorial>
            {_("This may take a moment while updates complete.")}
          </Content>
```